### ISSUE TITLE
ktlint reports "Internal Error (import-ordering)" when using kotlin_imports_layout=idea in .editorconfig

### ISSUE THREAD

**Issue by jai** [pinterest/ktlint#826]
## Expected Behavior
ktlint should pass (exit code 0).

## Observed Behavior
ktlint reports "Internal Error (import-ordering)"

```
/android-app/presentation/apply-presentation/src/main/kotlin/com/honestbank/android/apply/presentation/personal/ApplyPersonalFragment.kt:0:0: Internal Error (import-ordering). Please create a ticket at https://github.com/pinterest/ktlint/issues (if possible, provide the source code that triggered an error) (cannot be auto-corrected)
```

## Steps to Reproduce
Create a .editorconfig file in the root of the project with these contents:
```
[*.{kt,kts}]
kotlin_imports_layout=idea
```

Then run ktlint - here are some examples of import sets that are being reported with the above error:

Example 1 - multiple imports, only 'import' section shown:
```
import android.os.Bundle
import com.honestbank.android.apply.presentation.R
import com.honestbank.android.apply.presentation.common.CityBottomSheetListPickerViewModel
import com.honestbank.android.apply.presentation.common.EducationStatusBottomSheetListPickerViewModel
import com.honestbank.android.apply.presentation.mobile_verify.ApplyMobileVerifyViewEvent
import com.honestbank.android.common.presentation.di.ViewModelProvidersWrapper
import com.honestbank.android.common.presentation.livedata.nonNull
import com.honestbank.android.common.presentation.livedata.observe
import com.honestbank.android.common.presentation.view.BaseFragment
import com.honestbank.android.common.presentation.view.FormFieldListener
import com.honestbank.android.common.presentation.view.FormFieldType
import com.honestbank.android.common.presentation.view.HasFormFieldListener
import com.honestbank.android.common.presentation.view.bottomSheetListPicker.BottomSheetListDialog
import com.honestbank.android.common.presentation.view.bottomSheetListPicker.BottomSheetListViewModel
import com.honestbank.android.common.presentation.view.updateView
import kotlinx.android.synthetic.main.fragment_apply_personal_layout.personalCityOfBirth
import kotlinx.android.synthetic.main.fragment_apply_personal_layout.personalEducation
import kotlinx.android.synthetic.main.fragment_apply_personal_layout.personalMother
import kotlinx.android.synthetic.main.fragment_apply_personal_layout.personalNavigateButton
import javax.inject.Inject
```

Example 2 - single import statement, entire file shown:
```
package com.honestbank.android.apply.presentation.personal

import com.honestbank.android.common.presentation.view.FormFieldViewState

data class ApplyPersonalViewState(
    val cityOfBirth: FormFieldViewState,
    val educationStatus: FormFieldViewState,
    val motherMaidenName: FormFieldViewState
)
```

## Your Environment
* Version of ktlint used: 0.37.2

* Name and version (or code for custom task) of integration used (Gradle plugin, Maven plugin, command line, custom Gradle task): Gradle plugin `id "org.jlleitschuh.gradle.ktlint" version 9.2.1` and ktlint on the command line (macOS).

* Version of Gradle used (if applicable): 6.5.1

* Operating System and version: macOS 10.15.6

**Comment by romtsn** [comment]
Another question - are you building your project with kotlin 1.4 (rc, m1-m3)?

**Comment by romtsn** [comment]
I tried to reproduce it, but no luck - I assume you are building with kotlin-1.4 most likely.. if not, then I'd try to run ktlint via cli (that's what I did) to check whether there's a problem with the gradle plugin

**Comment by jai** [comment]
Kotlin version:
`kotlin_version = "1.3.72"`

I'll run on CLI and get back to you shortly.

**Comment by romtsn** [comment]
@jai any luck?

**Comment by jai** [comment]
Just tried the CLI, getting the same error/s:

```bash
android-app on  master [!+] 
➜ ktlint --format
/Users/jai/code/android-app/analytics/core-analytics/src/main/kotlin/com/honestbank/android/core/analytics/ApplyContactAnalyticsImpl.kt:0:0: Internal Error (import-ordering). Please create a ticket at https://github.com/pinterest/ktlint/issues (if possible, provide the source code that triggered an error) (cannot be auto-corrected)
/Users/jai/code/android-app/analytics/core-analytics/src/main/kotlin/com/honestbank/android/core/analytics/ApplyEmailAnalyticsImpl.kt:0:0: Internal Error (import-ordering). Please create a ticket at https://github.com/pinterest/ktlint/issues (if possible, provide the source code that triggered an error) (cannot be auto-corrected)
/Users/jai/code/android-app/analytics/core-analytics/src/main/kotlin/com/honestbank/android/core/analytics/ApplyEmailVerifyAnalyticsImpl.kt:0:0: Internal Error (import-ordering). Please create a ticket at https://github.com/pinterest/ktlint/issues (if possible, provide the source code that triggered an error) (cannot be auto-corrected)
```

**Comment by romtsn** [comment]
Could you provide the full contents of any of these files? (e.g. `ApplyContactAnalyticsImpl `)

**Comment by jai** [comment]
```kotlin
package com.honestbank.android.core.analytics

import com.honestbank.android.core.analytics.contracts.ApplyContactAnalytics
import com.honestbank.android.core.analytics.contracts.HonestAnalytics

class ApplyContactAnalyticsImpl(analytics: HonestAnalytics) :
    BaseApplyAnalyticsImpl(analytics),
    ApplyContactAnalytics {

    override val screenName: String = "apply.contact"
}
```

**Comment by romtsn** [comment]
@jai I've just tried your given example on a freshly created project and it works just fine for me. I assume something's wrong with your editorconfig - could you run the same executable but with `--debug` option? This should give you a detailed exception message

**Comment by jai** [comment]
Yeah so it worked fine today. I'm guessing my environment is dirty... been having some network issues with other libs as well.

I'll try and replicate again for my own benefit but yeah I'll close this issue I guess.
